README.md

```diff
@@ -85,19 +87,17 @@ First login to your balena account via `balena login`, and then create a new
 fleet:
 
 ```shell
-$ balena fleet create --type fincm3 DebugApp
-Fleet created: unicorn/debugapp (fincm3, id 1853079)
+$ balena fleet create --type raspberrypi4-64 DebugApp
+Fleet created: slug "ryanh/debugapp2", device type "raspberrypi4-64"
 ```
 
-Now provision a device, either by downloading a *development* image from the
-Dashboard, or by flashing via the command line (note that currently,
-balenaEtcher must be running to enable discovery of the balenaFin from balena
-CLI):
+Now provision a device by downloading and flashing a _development_ image from the
+Dashboard (via Etcher), or by flashing via the command line.
 
 ```shell
-$ balena os download fincm3 --version 2.80.3+rev1.dev --output balena-debug.img
-Getting device operating system for fincm3
-The image was downloaded successfully
+$ balena os download raspberrypi4-64 --version "2022.7.0.dev" --output balena-debug.img
```

Comment:
Are ESR releases available to free accounts?

Reply:
They are not, correct @klutchell . balena agents going through the masterclass will/should have "Team Member" accounts, so they have access to ESR releases. I also wanted a chance to plug them a bit. Do you think it would be a good idea to have a **Note** explaining and linking to ESR docs, perhaps with the caveat that for free accounts, they will need to select a different balenaOS release? None of the rest of my updates *require* ESR, so following along from other DT's and releases should be similar or nearly identical.

Reply:
Resolved with note in next commit